static/roles.js

```diff
@@ -43,7 +43,8 @@ const Roles =
         if (!this.heroes || !this.selfHeroes.length) return [];
         const roles = new Map();
         for (const sh of this.selfHeroes) {
-          for (const role of this.heroes[sh.hero_id].roles) {

```

Comment:
This no longer shows str/agi/int, ranged/melee on the UI.

Reply:
True. That was intentional.  I feel like those aren't roles.  The data is still there for a separate graph.

Reply:
Added back with following commit.